src/test/resources/testng_suites/web-saucedemo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="saucedemo test">
+    <test name="test login">
+        <classes>
+            <class name="com.solvd.meta.carina.LoginTest"/>
```

Comment:
Lets also specify method names, not just class. Update like this all classes here
<test name="Product test">
        <classes>
            <class name="com.solvd.meta.carina.CheckProductDetailScreenTest">
                <methods>
                    <include name="checkDetailsTest"/>
                </methods>
            </class>
        </classes>
    </test>